README.md

```diff
@@ -1,7 +1,12 @@
 # Fluke
-Fluke is a discord-like app written in Rust
+### Fluke is a discord-like app written in Rust
 
-***
+Make sure you set: `rustup target add wasm32-unknown-unknown`
+
+Files are served from backup/static, to build files run 
+`cd frontend`
+`trunk build --release` 
+or run `dev/serve.sh`
```

Comment:
shouldn't be in BE readme either

Reply:
Disagree, there the serve.sh will work from root, but if you try to cargo run in backend it wont have anything to serve unless you run trunk serve from frontend first